apps/challenges/templates/rabbitmq_not_reloaded.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
We can use same template here. Just pass some context variable to maintain your checks.